web/tsconfig.json

```diff
@@ -23,6 +23,6 @@
       "@/*": ["./*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts", "components/chat/chat-content"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts", "./components/chat/chat-content"],
```

Comment:
Why is it necessary to set up this directory separately? @Aralhi

Reply:
thanks, remove it.